docs/src/meshes/mesh_data_meshes.md

```diff
@@ -0,0 +1,125 @@
+`Globals1D, Globals2D, Globals3D` in the Matlab codes from "Nodal Discontinuous Galerkin Methods"
+by Hesthaven and Warburton (2007).
+
+In general, we use the following code conventions:
+* variables such as `r, s,...` and `x, y,...` correspond to values at nodal interpolation points.
+* variables ending in `q` (e.g., `rq, sq,...` and `xq, yq,...`) correspond to values at volume
+quadrature points.
+* variables ending in `f` (e.g., `rf, sf,...` and `xf, yf,...`) correspond to values at face
+quadrature points.
+* variables ending in `p` (e.g., `rp, sp,...`) correspond to "plotting" points, which are usually
+a fine grid of equispaced points.
+* `V` matrices correspond to interpolation matrices from nodal interpolation points, e.g., `Vq`
+interpolates to volume quadrature points, `Vf` interpolates to face quadrature points.
+* geometric quantities in `MeshData` are stored as matrices of dimension
+``\text{number of points per element } \times \text{number of elements}``.
+
+Quantities in `rd::RefElemData`:
+* `rd.Np, rd.Nq, rd.Nf`: the number of nodal interpolation points, volume quadrature points, and
+face quadrature points on the reference element, respectively.
+* `rd.Vq`: interpolation matrices from values at nodal interpolation points to volume quadrature points
+* `rd.wq`: volume quadrature weights on the reference element
+* `rd.Vf`: interpolation matrices from values at nodal interpolation points to face quadrature points
+* `rd.wf`: a vector containing face quadrature weights on the reference element
+* `rd.M`: the quadrature-based mass matrix, computed via `rd.Vq' * diagm(rd.wq) * rd.Vq`.
+* `rd.Pq`: a quadrature-based ``L^2`` projection matrix `rd.Pq = rd.M \ rd.Vq' * diagm(rd.wq)`
+which maps between values at quadrature points and values at nodal points.
+* `Dr, Ds, Dt` matrices are nodal differentiation matrices with respect to the ``r,s,t`` coordinates,
+e.g., `Dr*f.(r,s)` approximates the derivative of ``f(r,s)`` at nodal points.
+
+Quantities in `md::MeshData`:
+* `md.xyz` is a tuple of matrices `md.x`, `md.y`, `md.z`, where column `e` contains coordinates of
+physical interpolation points.
+* `md.xyzq` is a tuple of matrices `md.xq`, `md.yq`, `md.zq`, where column `e` contains coordinates
+of physical quadrature points.
+* `md.rxJ, md.sxJ, ...` are matrices where column `e` contains values of
+``J\frac{\partial r}{\partial x}``, ``J\frac{\partial s}{\partial x}``, etc. at nodal interpolation
+points on the element `e`.
+* `md.J` is a matrix where column `e` contains values of the Jacobian ``J`` at nodal interpolation points.
+* `md.Jf` is a matrix where column `e` contains values of the face Jacobian (e.g., determinant of
+the geometric mapping between a physical face and a reference face) at face quadrature points.
+* `md.nxJ, md.nyJ, ...` are matrices where column `e` contains values of components of the unit
+normal scaled by the face Jacobian `md.Jf` at face quadrature points.
```

Comment:
```suggestion
* `rd.Np, rd.Nq, rd.Nf`: the number of nodal interpolation points, volume quadrature points, and
  face quadrature points on the reference element, respectively.
* `rd.Vq`: interpolation matrices from values at nodal interpolation points to volume quadrature points
* `rd.wq`: volume quadrature weights on the reference element
* `rd.Vf`: interpolation matrices from values at nodal interpolation points to face quadrature points
* `rd.wf`: a vector containing face quadrature weights on the reference element
* `rd.M`: the quadrature-based mass matrix, computed via `rd.Vq' * diagm(rd.wq) * rd.Vq`.
* `rd.Pq`: a quadrature-based ``L^2`` projection matrix `rd.Pq = rd.M \ rd.Vq' * diagm(rd.wq)`
  which maps between values at quadrature points and values at nodal points.
* `Dr, Ds, Dt` matrices are nodal differentiation matrices with respect to the ``r,s,t`` coordinates,
  e.g., `Dr*f.(r,s)` approximates the derivative of ``f(r,s)`` at nodal points.

Quantities in `md::MeshData`:
* `md.xyz` is a tuple of matrices `md.x`, `md.y`, `md.z`, where column `e` contains coordinates of
  physical interpolation points.
* `md.xyzq` is a tuple of matrices `md.xq`, `md.yq`, `md.zq`, where column `e` contains coordinates
  of physical quadrature points.
* `md.rxJ, md.sxJ, ...` are matrices where column `e` contains values of
  ``J\frac{\partial r}{\partial x}``, ``J\frac{\partial s}{\partial x}``, etc. at nodal interpolation
  points on the element `e`.
* `md.J` is a matrix where column `e` contains values of the Jacobian ``J`` at nodal interpolation points.
* `md.Jf` is a matrix where column `e` contains values of the face Jacobian (e.g., determinant of
  the geometric mapping between a physical face and a reference face) at face quadrature points.
* `md.nxJ, md.nyJ, ...` are matrices where column `e` contains values of components of the unit
  normal scaled by the face Jacobian `md.Jf` at face quadrature points.
```
I'm no native speaker, but the inconsistent use of a `.` at the end of each item looks a bit weird to me. Is that done on purpose?

Reply:
No, it's accidental. I'll add periods for each sentence to be consistent.